packages/remix-react/data.ts

```diff
@@ -1,9 +1,11 @@
+import type { FormMethod as FormMethodRR } from "react-router-dom";
+
 import invariant from "./invariant";
 import type { Submission } from "./transition";
 
 export type AppData = any;
 
-export type FormMethod = "get" | "post" | "put" | "patch" | "delete";
+export type FormMethod = FormMethodRR;
```

Comment:
This is something we need to do a pass on still - probably have _lots_ of types that can either be deleted or just re-export something from RR to keep type-API compatibility